script.js

```diff
@@ -0,0 +1,172 @@
+  foodName.innerText = recipeObject.title;
+  descriptionContainer.appendChild(foodName);
+
+  const ingredientsTitle = document.createElement("p");
+  ingredientsTitle.innerText = "Ingradients:";
+  ingredientsTitle.style.textDecoration = "underline";
+  descriptionContainer.appendChild(ingredientsTitle);
+
+  const ingredientsList = document.createElement("ul");
+  ingredientsList.classList.add("ingredientsList");
+
+  for (let i = 0; i < recipeObject.ingredients.length; i++) {
+    let listItem = document.createElement("li");
+    let listItemName = document.createElement("div");
+    let listItemAmount = document.createElement("div");
+    listItem.appendChild(listItemName);
+    listItem.appendChild(listItemAmount);
+    listItemName.innerText = recipeObject.ingredients[i].NAME;
+    listItemAmount.innerText = recipeObject.ingredients[i].AMOUNT;
+    ingredientsList.appendChild(listItem);
+  }
+  descriptionContainer.appendChild(ingredientsList);
+
+  const descriptionTitle = document.createElement("p");
+  descriptionTitle.classList.add("recipeDescription");
+  descriptionTitle.innerText = "Description:";
+  descriptionTitle.style.textDecoration = "underline";
+  const description = document.createElement("p");
+
+  description.innerText = recipeObject.description;
+  descriptionContainer.appendChild(descriptionTitle);
+  descriptionContainer.appendChild(description);
+}
+
+//**************** Add new recipe Button *****************/
+const addRecipeBtn = document.getElementById("addRecipeBtn");
+addRecipeBtn.addEventListener("mouseover", function (e) {
+  addRecipeBtn.classList.add("btnMouseOver");
+});
+addRecipeBtn.addEventListener("mouseout", function (e) {
+  addRecipeBtn.classList.remove("btnMouseOver");
+});
+
+//**************** Form visible  *****************/
+
+function addRecipeClick() {
+  const formShow = document.getElementById("formItemsContainer");
+  formShow.style.display = "flex";
+  formShow.scrollIntoView();
+}
+
+function cancelBtnClicked() {
+  const formShow = document.getElementById("formItemsContainer");
+  formShow.style.display = "none";
+}
+
+//**************** Add new recipe  *****************/
+
+function saveBtnClicked() {
+  recipeCounter = recipeCounter + 1;
+
+  let titleValue = document.getElementById("title").value;
+  let pictureUrlValue = document.getElementById("picture").value;
+  let descriptionValue = document.getElementById("descriptionArea").value;
+  let ingredientsList = [];
+  for (let i = 0; i <= ingredientNumber; i++) {
+    let nameValue = document.getElementById("ingName" + i).value;
+    let amountValue = document.getElementById("ingAmount" + i).value;
+    let newIng = { NAME: nameValue, AMOUNT: amountValue };
+    ingredientsList.push(newIng);
+  }
+
+  const newRecipe = {
+    id: recipeCounter,
+    title: titleValue,
+    picture_url: pictureUrlValue,
+    ingredients: ingredientsList,
+    description: descriptionValue,
+  };
+  recipes.push(newRecipe);
```

Comment:
It would be beneficial to add some validation before saving the new recipe. For example, I could add an empty recipe.
You don't have to do it now but think about it for the coming weeks.